modules/comment/comment.resolver.js

```diff
@@ -25,6 +25,19 @@ const commentsQuery = {
       ];
     }
   },
+
+  getAllCommentsByUser: async (parent, args) => {
+    try {
+      return await commentsService.getAllCommentsByUser(args.userEmail);
```

Comment:
I think we need here to have a check that valid email is provided via `joi-validator`

Reply:
This method is called on admin panel and shows comments for already existing user and this kind of validation is provided in `validate-user` file. Are there any other reasons for this check?